assets/sass/components/global/_googlesitekit-badge-with-tooltip.scss

```diff
@@ -16,14 +16,10 @@
  * limitations under the License.
  */
 
-.googlesitekit-audience-segmentation-partial-data-badge {
+.googlesitekit-badge-with-tooltip {
 	background-color: $c-utility-warning-container;
-	border-radius: $br-lg;
 	color: $c-yellow-y-600;
 	display: inline-flex;
-	font-size: $fs-label-sm;
-	font-weight: $fw-medium;
-	letter-spacing: $ls-xs;
 	line-height: 16px;
 	padding: 6px 10px;
```

Comment:
It looks like I originally missed it, but if you take a look, the badge in the selection panel has a vertical padding of 4px, whereas when used as a partial data badge outside the selection panel, it has a vertical padding of 6px. We'll have to override this in the selection panel stylesheet to enforce a vertical padding of 4px.